listenbrainz/webserver/views/stats_api.py

```diff
@@ -721,8 +721,9 @@ def _get_mbids_from_msids(artist_msids: list) -> list:
     request_data = [{"artist_msid": artist_msid} for artist_msid in artist_msids]
     artist_mbids = []
     try:
-        result = requests.post(
-            "{}/artist-credit-from-artist-msid/json".format(current_app.config['LISTENBRAINZ_LABS_API_URL']), json=request_data)
+        result = requests.post("{}/artist-credit-from-artist-msid/json?count={}"
```

Comment:
Instead of using string formatting, we should pass count into the params parameter of requests.post.